cedar-lean/Cedar/Thm/Authorization/Authorizer.lean

```diff
@@ -310,80 +300,37 @@ theorem mapM_asEntityUID_of_uid {uids : List EntityUID} :
   case nil =>
     rw [List.map_nil, List.mapM_nil]
     simp [pure, Except.pure]
-  case cons x xs h_ind =>
+  case cons x xs ih =>
     rw [List.map_cons, List.mapM_cons]
-    simp [pure, Except.pure, asEntityUID_of_uid, h_ind]
+    simp [pure, Except.pure, asEntityUID_of_uid, ih]
 
-/--
-  A generic lemma about the behavior of List.mapM' in the Except monad
--/
-theorem mapM'_ok_iff_f_ok_on_all_elements {f : α → Except ε β} {list : List α} :
```

Comment:
Moved to Thm/Data/List.lean, reformulated in terms of `mapM` instead of `mapM'`, and renamed `isOk_mapM`

Reply:
Superseded by https://github.com/cedar-policy/cedar-spec/pull/305#discussion_r1587719350